lib/activerecord-import/import.rb

```diff
@@ -502,7 +501,7 @@ def import_without_validations_or_callbacks( column_names, array_of_attributes,
         end
       else
         values_sql.each do |values|
-          ids << connection.insert(insert_sql + values)
+          connection.execute(insert_sql + values)
```

Comment:
This block is for adapters that don't support bulk import, like old version of SQLite. I don't think it would need to change.